awx/main/models/credential/__init__.py

```diff
@@ -1008,21 +1008,23 @@ def create(self):
 ManagedCredentialType(
     namespace='tower',
     kind='cloud',
-    name=ugettext_noop('Ansible Tower'),
+    name=ugettext_noop('Red Hat Ansible Automation Platform'),
     managed_by_tower=True,
     inputs={
         'fields': [
             {
                 'id': 'host',
-                'label': ugettext_noop('Ansible Tower Hostname'),
+                'label': ugettext_noop('Red Hat Ansible Automation Platform'),
                 'type': 'string',
-                'help_text': ugettext_noop('The Ansible Tower base URL to authenticate with.'),
+                'help_text': ugettext_noop('Red Hat Ansible Automation Platform base URL to authenticate with.'),
             },
             {
                 'id': 'username',
                 'label': ugettext_noop('Username'),
                 'type': 'string',
-                'help_text': ugettext_noop('The Ansible Tower user to authenticate as.' 'This should not be set if an OAuth token is being used.'),
+                'help_text': ugettext_noop(
+                    'Red Hat Ansible Automation Platform user to authenticate as.' 'This should not be set if an OAuth token is being used.'
```

Comment:
I would change this to "user id" for clarity

Reply:
Made the changes, will be pushed in next commit.

Reply:
maybe `username`? Don't want to specify a PK here. (I don't think anyone would do that, but you never know...)

Reply:
that's totally fair ^ I'm not tied to User ID in any way, I vote for clarity